<launch>

  <arg name="config" default="true" />
  <arg unless="$(arg config)" name="command_args" default="" />
  <arg     if="$(arg config)" name="command_args" default="-d $(find moveit_recorder)/launch/recordit.rviz" />

  <node name="$(anon rviz)" pkg="rviz" type="rviz" respawn="false" args="$(arg command_args)" output="screen">
    <rosparam command="load" file="$(find pr2_moveit_config)/config/kinematics.yaml"/>
  </node>

  <arg name="save_dir" default="$(find moveit_recorder)/test/view.bag" />

  <node name="viewpoint_recorder" pkg="moveit_recorder" type="viewpoint_recorder_node" 
    args="--save_dir $(arg save_dir)" respawn="false" output="screen">
  </node>
  
</launch>
